MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1152 @@
+</td>
+</tr>
+<tr>
+<td>
+RadioGroup
+</td>
+<td>
+Enum and List type property.
+</td>
+<td>
+{{'[RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+</table>
+
+## To change the editor for any data type
+
+By default, the editors will be loaded based on the datatype. To change the editor for any type, use the `RegisterEditor` method and specify the type and editor.
+
+{% tabs %}
+{% highlight C# %}
+
+    this.dataForm.RegisterEditor(typeof(string), DataFormEditorType.MultilineText);
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the `MultilineText` editor will be loaded for the string type instead of `Text` editor.
+
+## Changing the editor for property
+
+To change the editor for any property, use the `RegisterEditor` method and specify the property name and editor.
+
+{% tabs %}
+{% highlight c# %}
+
+    this.dataForm.RegisterEditor("On", DataFormEditorType.Switch);
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the Switch editor will be loaded for the `On` property (bool type) instead of `CheckBox` editor.
+
+## Text editor
+
+In the text editor, the [Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0) is loaded.
+
+Text editor have Keyboard type change by using `Keyboard` property and you can set maximum allowed length of input by using `MaxLength` property.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.Text)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Multiline Text editor
+
+In the `MultilineText` editor, the [Editor](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-7.0) is loaded.
+
+And `MultilineText` editor height will auto expand/reduce based on the line wraps in editor , which allowing text to be readable without scrolling the editor.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.MultilineText)]
+    public string Address { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Password editor
+
+In the password editor, the [Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0) is loaded.
+
+{% tabs %}
+{% highlight C# %}
+
+    [Display(ShortName = "Transaction password", Prompt = "Enter password")]
+    [DataType(DataType.Password)]
+    public string Password { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Date editor
+
+In the date editor, the [DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0) will be loaded.
+
+In [DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0), the default date value is current date displayed. You can also adding nullable `DateTime` data type for the date picker property in data form, which allows you to set the current date and display current date in date editor. 
```

Comment:
In the date editor, the default date value will be the current date. You can also adding nullable `DateTime` data type for the date picker property in data form, which allows you to set the current date and display the current date in the date editor.

Reply:
content updated